src/main/java/frc/robot/subsystems/shooter/Flywheel.java

```diff
@@ -47,13 +51,17 @@ public void setRollerSpeed(double speed) {
     Logger.recordOutput("Roller/DutyCycleOut", speed);
   }
 
+  /** updates and logs inputs periodically */
```

Comment:
methods with @Override shouldn't be given docstrings, they already have them (try hovering over them)

Reply:
Ok Ishan told me to docstring for override